### DISCUSSION TITLE
Show all steps - easier interaction

### DISCUSSION THREAD

**Discussion by DennisNissenUX** [Budibase/budibase#7528] [Ideas]
Hey guys,

Yesterday I designed a multi step form and had an idea that I would like to discuss.
In itself it works well to navigate from step to step, but in my opinion it lacks one or two more settings. In the form I designed, it makes sense to be able to display all entered data at the end as the last step. Unfortunately, I have not found this setting option in the steps. It seems to be only possible to display one step at a time.
I implemented the form not with steps but with state variables and conditional UI, so it was no problem to display all steps (or only a certain selection of steps).
Have I not understood the component Form Steps correctly or do you think it would be useful to adjust the step navigation so that you can also display multiple steps (or all).

Looking forward to your comments.

![Bildschirmfoto 2022-08-30 um 11 31 03](https://user-images.githubusercontent.com/105709029/187402377-4b395f56-a742-4028-91c0-7a8a952bc1b7.png)

**Comment by melohagan** [comment]
Hey Dennis, this is an interesting idea, but I think it might not be necessary. 

Because you have access to the form field bindings, you could always display a summary page on the last step. You could add another form with disabled fields that have the *Default value* set to the corresponding form fields. 

Alternatively you could display the form field values (via the bindings) in a nicer way such as in a card component. 
If they want to change anything you have the **Specific step** option that could facilitate that.
